src/lib/Controller/api/receiver.c

```diff
@@ -191,145 +191,162 @@ void wb_receiver_init(WbDevice *d) {
 
 void wb_receiver_enable(WbDeviceTag tag, int sampling_period) {
   if (sampling_period < 0) {
-    fprintf(stderr, "Error: wb_receiver_enable() called with negative sampling period.\n");
+    fprintf(stderr, "Error: %s() called with negative sampling period.\n", __FUNCTION__);
```

Comment:
Unfortunately, this introduces a non-homogeneity with the other devices, ideally, we should change this in the whole C API.

Reply:
Yes, that would be a good practice. I can do it in this PR.

Reply:
That would be nice, thank you.